src/coding-guidelines/expressions.rst

```diff
@@ -81,3 +81,91 @@ Expressions
          }
 
          fn with_base(_: &Base) { ... }
+
+.. guideline:: Do not use builtin arithmetic expressions
+   :id: gui_7y0GAMmtMhch
+   :category: required
+   :status: draft
+   :release: latest
+   :fls: fls_Q9dhNiICGIfr
+   :decidability: decidable
+   :scope: module
+   :tags: numerics
+
+   This guideline applies when an ArithmeticExpression is used with operands whose types evaluates integer or
+   floating-point types.
```

Comment:
You know, I'm not sure floating-point types fit here.

* One one hand, the `f32` and other such types are defined to be [compliant with a specific IEEE standard](https://doc.rust-lang.org/std/primitive.f32.html).
* On the other hand, neither looking at that type's methods nor doing [a search for `checked_` methods in stdlib](https://doc.rust-lang.org/std/index.html?search=checked) I've been able to find anything like the safe/checked operations we have for integer types. I suspect they do not exist for floats.

Reply:
```suggestion
   This guideline applies when an `ArithmeticExpression` is used with operands of integer or
   floating-point type.
```

Reply:
I think I agree I would separate integer from floating point types.

Reply:
You are correct, the float point types don't fit here. Removed, and resolved formatting/grammar in 8d2392d4a942207d325a842863c6c8e4f9d5756f